doc/whats_new.rst

```diff
@@ -41,7 +41,7 @@ Changelog
 
 - Add ability to add tonic inputs to cell types with :func:`hnn_core.Network.add_tonic_input`, by `Mainak Jas`_ in `#209 <https://github.com/jonescompneurolab/hnn-core/pull/209>`_
 
-- Modify plot_spikes_raster() to display individual gids, by `Nick Tolley`_ in `#231 <https://github.com/jonescompneurolab/hnn-core/pull/231>`_
+- Modify plot_spikes_raster() to display individual cells, by `Nick Tolley`_ in `#231 <https://github.com/jonescompneurolab/hnn-core/pull/231>`_
```

Comment:
I slowly give you more info ;-)

you can actually link to the API documentation here by saying :func:`hnn_core.viz.plot_spikes_raster`